src/fibaroAccessory.ts

```diff
@@ -529,6 +541,7 @@ export class FibaroAccessory {
     service.isClimateZone = (IDs.length >= 3 && IDs[2] === 'CZ') ? true : false;
     service.isHeatingZone = (IDs.length >= 3 && IDs[2] === 'HZ') ? true : false;
     service.isOpenCloseOnly = (IDs.length >= 3 && IDs[2] === 'OPENCLOSEONLY') ? true : false;
+    service.isPM2_5Sensor = (IDs.length >= 3 && IDs[2] === 'PM2_5') ? true : false;
```

Comment:
Need to specify service.isPM2_5Sensor? Is it used anywhere after that?

Reply:
Ok it is in getAirQuality function but is it required there?

Reply:
Yes, it is used to check which type of air quality sensor it is.